pycbc/types/array.py

```diff
@@ -615,6 +615,19 @@ def conj(self):
     def squared_norm(self):
         """ Return the elementwise squared norm of the array """
 
+
+    # Adding the @_checkother decorator here raises a bizarre failure when
+    # calling weave. KeyError: 'V'. Why??
```

Comment:
I don't think so. checkother seems to actually be checkothers as it loops over arguments. The issue seems to be that it returns other._data for all inputs *except* self. Then Array._data.data will be some strange type. However, convert (which operates on self) returns an Array. I guess I could work around this, but it seems somewhat counterintuitive. Really the thing I want from this is the precision conversion on scalars!